net-libs/zeromq/zeromq-4.2.3.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="A lightweight messaging kernel"
+DESCRIPTION="The ZeroMQ lightweight messaging kernel is a library which \
+extends the standard socket interfaces with features traditionally provided by\
+ specialised messaging middleware products. ZeroMQ sockets provide an \
+abstraction of asynchronous message queues, multiple messaging patterns, \
+message filtering (subscriptions), seamless access to multiple transport \
+protocols and more."
+HOMEPAGE="http://www.zeromq.org/"
+COPYRIGHT="2007-2014 iMatix Corporation
+	2009-2011 250bpm s.r.o.
+	2010-2011 Miru Limited
+	2011 VMware, Inc.
+	2012 Spotify AB
+	2013 Ericsson AB
+	2014 AppDynamics Inc.
+	2015 Google, Inc.
+	2015-2016 Brocade Communications Systems Inc."
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/zeromq/libzmq/releases/download/v$portVersion/zeromq-$portVersion.tar.gz"
+CHECKSUM_SHA256="8f1e2b2aade4dbfde98d82366d61baef2f62e812530160d2e6d0a5bb24e40bc0"
+SOURCE_URI_2="https://raw.githubusercontent.com/zeromq/cppzmq/a604af9/zmq.hpp#noarchive"
```

Comment:
cppzmq provides a seperate archive at:
https://github.com/zeromq/cppzmq
I've used the latest push for zmq.hpp now (not sure if it's needed to build a seperate package for it?)